File: mpp_proiect_java/TriatlonModel/src/main/java/domain/Proba.java
package domain;

public class Proba extends Entity<Long>{
    /***
     * tipProba - inot/ciclism/alergare
     * distanta - distanta pentru o proba
     */
    private String tipProba;
    private Integer distanta;

    public Proba(Long idProba,String tipProba,Integer distanta){
        setId(idProba);
        this.tipProba = tipProba;
        this.distanta = distanta;
    }

    public String getTipProba() {
        return tipProba;
    }

    public void setTipProba(String tipProba) {
        this.tipProba = tipProba;
    }

    public Integer getDistanta() {
        return distanta;
    }

    public void setDistanta(Integer distanta) {
        this.distanta = distanta;
    }

    @Override
    public String toString() {
        return "Proba{" +
                "tipProba='" + tipProba + '\'' +
                ", distanta='" + distanta + '\'' +
                '}';
    }
}

File: mpp_proiect_CSharp/triatlon/model/Proba.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace triatlon.domain
{
    [Serializable]
    public class Proba : Entity<long>
    {
        public long Id { get; set; }
        public string tipProba { get; set; }

        public int distanta { get; set; }
        public Proba()
        {

        }

        public Proba(long idProba,string tipProba1,int distanta1)
        {
            this.Id = idProba;
            this.tipProba = tipProba1;
            this.distanta = distanta1;
        }

        public override string ToString()
        {
            return "Proba - tip proba: " + tipProba + '|' +
                "distanta=" + distanta;
        }
    }
}
